coderd/workspaces.go

```diff
@@ -165,10 +165,37 @@ func (api *API) workspaceByOwnerAndName(rw http.ResponseWriter, r *http.Request)
 	owner := httpmw.UserParam(r)
 	workspaceName := chi.URLParam(r, "workspacename")
 
+	var (
+		includeDeletedStr = r.URL.Query().Get("include_deleted")
```

Comment:
Suggestion: for consistency, rename this query parameter to `deleted` (see `workspaces.go:41`)

```suggestion
		includeDeletedStr = r.URL.Query().Get("deleted")
```

Reply:
@johnstcn this actually does something a little different than `deleted` (or at least, I want it to). The `deleted` for `/workspace` is not inclusive - if you add that param on a workspace that is not deleted, you get an error. Here, I want to get a record back if I pass through `include_deleted`, regardless of whether or not the record is actually deleted. Does that make sense? Happy to rename if it still seems off to you.

Reply:
Ah, that makes sense! Disregard then.